vendors/polysense/drivers/WxS8800-362021/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "WxS8800-362021",
+  "version": "1.0.0",
+  "description": "The WxS8800-362021 is a LoRaWAN Sound/Noise Smart Sensor & RTU (Remote Terminal Unit) 2-in-1 Terminal.",
```

Comment:
no need for this field, or it should be a description of the driver package not the device.